juntagrico/admins/subscription_type_admin.py

```diff
@@ -7,7 +7,7 @@
 class SubscriptionTypeAdmin(SortableAdminMixin, RichTextAdmin):
     list_display = ['__str__', 'price', 'required_assignments',
                     'required_core_assignments', 'visible']
-    exclude = ['trial']
+    exclude = ['trial', 'trial_days']
```

Comment:
```suggestion
    exclude = ['trial']
```
In the meantime the trial_days are used to display in the signup pages that the subscription only lasts for x days and does not renew.